vietanh2810/Sorting-Visualizer
# Adding github Action CI/CD

## Summary

This merge request resolves issue #9 by creating script for GitHub actions

## Changes Made

- add a yml file

## Verification Steps

To verify that it is working correctly, follow these steps:

1. On push or pull request in main branch, the code will be triggered to build and deploy to GitHub page

## Reviewers

@vietanh2810
